components/authz-service/server/v2/project.go

```diff
@@ -151,10 +160,15 @@ func (s *ProjectState) CreateProject(ctx context.Context,
 
 func (s *ProjectState) UpdateProject(ctx context.Context,
 	req *api.UpdateProjectReq) (*api.UpdateProjectResp, error) {
+
+	err := confirmRequiredIDandName(req.Id, req.Name, "project")
```

Comment:
[nit] It looks weird, but the canonical casing here would be this:
```suggestion
	err := confirmRequiredIDAndName(req.Id, req.Name, "project")
```

...but I'm down to turning a blind eye 🙈

Reply:
this does seem like an opportunity conceding that the "right" way is not always the most readable, huh

Reply:
Yeah probably. Also naming is hard 😉